Views/AccountPanel.php

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Account Panel</title>
+    <style>
```

Comment:
Styles needs to be extracted into multiple classes within the stylesheet file instead of being here